warehouse/forklift/legacy.py

```diff
@@ -1363,8 +1361,8 @@ def file_upload(request):
     # Dispatch our task to sync this to cache as soon as possible
     request.task(sync_file_to_cache).delay(file_.id)
 
-    # Return any warnings that we've accumulated as the response body.
-    return HTTPOk(body="\n".join(warnings))
+    # Return successful response
+    return HTTPOk()
```

Comment:
Seems like a valid cleanup since nothing seems to append to `warnings` anymore. I'm not sure how this was used in the past, but feels like removing it will lose the (seemingly) undocumented method of notifying clients.

Reply:
Good point, will restore with some inline docs